.github/CONTRIBUTING.md

```diff
@@ -140,14 +140,15 @@ Also check the other examples:
 ```bash
 cd examples/EXAMPLENAME
 yarn install
-yarn start
+npm start
 ```
 
 Releases are managed by GitHub Actions, here’s an overview of the process to release a new Uppy version:
 
 * Run `yarn release` on your local machine.
 * Follow the instructions and select what packages to release.
 * Before committing, check if the generated files look good.
+* When asked to edit the next CHANGELOG, only include changes related to the package(s) you selected for release.
```

Comment:
Something worth noting is that if we skip some packages during a release, all the changes on those packages are "lost", meaning my tool won't know there are unreleased changes in those packages (it fetches all changes since latest release from GitHub, so commits that landed before are considered already released). I'm not sure if there is a way to work around that – I've thought of adding commit hashes to the CHANGELOG, and make the tool parse CHANGELOG to figure out which commits are unreleased. but I don't think it would scale if the tool has to check for 9k commits each time we want to do a release.

So maybe we should instead encourage releasers to not skip packages, unless the changes are unrelated to the npm package.

Reply:
You could encourage it but it's a valid use case to for instance just do a security release for companion and no other packages. Then you do need to remove all the things from the changelog that are from the other packages.

Reply:
I see, so we should always release every package that has any changes, and make sure that the main branch is always stable and ready for release?

Reply:
It seems I read this wrong the first time. If that's the case I think a big warning should be added to the instructions because it's important

Reply:
Should we say something about not skipping packages unless the changes are actually not touching anything related to the npm package?

Reply:
sure! i updated the comment